deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
```

Comment:
I would advise to move this deploy stuff and the gdb_test.sh somewhere else

Reply:
moved to deploy repo